code/game/objects/items/stacks/stack.dm

```diff
@@ -234,7 +234,9 @@
 	return
 
 /obj/item/stack/AltClick(mob/living/user)
-	if(user.incapacitated())
+	var/speciescheck = !ishuman(user) || !isdrone(user)
```

Comment:
All mobs have hands, at least defined, as per https://github.com/tgstation/tgstation/blob/master/code/modules/mob/mob_defines.dm#L82

You'd need to do something use procs `get_active_held_item()` (or `get_inactive_held_item()`, not sure which one precisely, @RemieRichards should be able to tell you, what with the mob hand rework and all that).